root/types/relationship.js

```diff
@@ -100,8 +100,8 @@ declare type RelationshipT = $ReadOnly<{
   +linkOrder: number,
   +linkTypeID: number,
   +source_id: number | null,
-  +source_type: string,
+  +source_type: CoreEntityTypeT,
   +target: CoreEntityT,
-  +target_type: string,
+  +target_type: CoreEntityTypeT,
```

Comment:
This seems sensible, but please amend the commit title "Change source/type_type on RelationshipT to CoreEntityTypeT" :)